sasjs/cli
# fix(job flags): --log, --output and new --status

## Issue

Closes #250 

## Intent

To update `--log` flag, `--output` flag's behavior and add new `--status` flag
++ handle absolute paths for `output`, `logfile` and `status`
++ handle `~` for win paths
++ use default context if not specified

## Implementation

Updated command utility:
- for `--log` flag updated to `--logFile` and alias are working i.e. `--log` and `-l`
- to have another flag `--status`
Update job execute code to have `status: success` in output file in case of `wait` and print on screen.

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
- [x] Any new functionality has been unit tested.
- [x] All unit tests are passing (`npm test`).
tested with unreleased adapter 
![image](https://user-images.githubusercontent.com/8914650/100032991-0bafd700-2e1b-11eb-944b-376cb412b4c3.png)
- [ ] All CI checks are green.
- [x] JSDoc comments have been added or updated.

## Steps to test
- checkout branch `issue250`
- `npm install`
- `npm link`
- verify with `npm -v` it should give you the path of local repository of sasjs-cli
_You are using a linked version of SASjs CLI running from sources at 1.0.0   {PATH}_
